src/i_video.c

```diff
@@ -1787,6 +1787,7 @@ void I_InitGraphics(void)
     {
         I_Error("Failed to initialize video: %s", SDL_GetError());
     }
+    SDL_SetHint("SDL_HINT_RENDER_BATCHING", "0");
```

Comment:
Just a note that this seems to fix https://github.com/fabiangreffrath/woof/issues/1685#issuecomment-2134358231 too. Well, at least until SDL3 removes the hint altogether. But that's something to worry about later I suppose.

Reply:
@MrAlaux How does this PR behave on your system (with or without exclusive fullscreen)?

Reply:
Reminder for everyone else: I'm running Windows 7 x64 on an i3-350M.

I ran the game at native resolution (1364x640) with Dynamic Resolution disabled (and FWIW, Music Volume set to 0) but otherwise default settings. I've no tooling to measure latency and whatnot, so this is all by eye.

In both the latest master and this PR, the game _maybe_ feels more responsive with Exclusive Fullscreen disabled, but I'm not completely sure about it. Even if, the difference is marginal.

Reply:
Okay thanks, I know your system is an edge case so I was curious about compatibility and it seems fine.